tcmur_device.c

```diff
@@ -276,7 +274,7 @@ int tcmu_acquire_dev_lock(struct tcmu_device *dev)
 	switch (ret) {
 	case TCMUR_LOCK_BUSY:
 		new_state = TCMUR_DEV_LOCK_LOCKING;
-		break;
+		return ret;
```

Comment:
Let's just remove all the TCMUR_LOCK_BUSY code since it is not used. I think without your patch it is buggy and with your patch we would hit a new bug where we leave the device blocked.

Reply:
Actually this code wouldn't be executed by any case.
Just removed it.